net/libfab/libfab.h

```diff
@@ -36,17 +36,27 @@
  * @{
  */
 
-// #define FI_MAJOR_VERSION 1
-// #define FI_MINOR_VERSION 1
-
-struct libfabric_domain_params {
+/**
+ *    Libfabric parameters.
+ * 
+ */
+struct libfab_dom_params {
```

Comment:
These struct members should be named as:
```
struct libfab_dom_params {
        struct fi_info    *ldp_fi;      /* Fabric info */
        struct fi_info    *ldp_hints;   /* Fabric info to configure flags */
        ...
```

Reply:
renamed struct and members